### ISSUE TITLE
Following example in README.md fails

### ISSUE THREAD

**Issue by jmwilkinson** [Blizzard/node-rdkafka#139]
I've been attempting to follow the example outlined by the README.

Unfortunately, I get the following error:

```
/home/jwilkinson/kafka-test/node_modules/node-rdkafka/lib/kafka-consumer.js:202
  this._errorWrap(this._client.subscribe(topics));
                               ^

Error: First parameter must be an array
    at Error (native)
    at KafkaConsumer.subscribe (/home/jwilkinson/kafka-test/node_modules/node-rdkafka/lib/kafka-consumer.js:202:32)
    at KafkaConsumer.<anonymous> (/home/jwilkinson/kafka-test/test.js:60:18)
    at emitOne (events.js:101:20)
    at KafkaConsumer.emit (events.js:188:7)
    at /home/jwilkinson/kafka-test/node_modules/node-rdkafka/lib/client.js:214:12
    at /home/jwilkinson/kafka-test/node_modules/node-rdkafka/lib/client.js:351:7
```

I am attempting to setup the `KafkaConsumer` as per the example in the docs:

```
let consumer = new Kafka.KafkaConsumer({
  'group.id': 'consumer-group',
  'metadata.broker.list': 'broker:9092',
}, {});


// Flowing mode
consumer.connect();

consumer
    .on('ready', function() {
        consumer.subscribe('topic1');

        // Consume from the librdtesting-01 topic. This is what determines
        // the mode we are running in. By not specifying a callback (or specifying
        // only a callback) we get messages as soon as they are available.
        consumer.consume();
    })
    .on('data', function(data) {
        console.log('Consumer received a message');
        // Output the actual message contents
        console.log(data.message.toString());
    });
```

Is this an issue with the docs, my setup, or the library?

**Comment by tallboy** [comment]
I believe as of `0.7.*` topics need to be passed in as an array.  Try changing `consumer.subscribe('topic1');` to `consumer.subscribe(['topic1']);` and see if that works.  I believe in this case the `README.md` is out of date with the latest consumer code.

**Comment by jmwilkinson** [comment]
Thanks @tallboy, it appears to be working. I think I'll leave the open as the docs should be changed.

**Comment by webmakersteve** [comment]
Fixed README with 5c80629434aad032f492233add80fac2b423dd7f and fixed example with #141
